src/content/desci/index.md

```diff
@@ -8,77 +8,267 @@ sidebar: true
 
-### IP ownership
+The web3 community is still in its early stages, and there is a lot of work to be done in order to realize the full potential of decentralized science. However, the potential benefits are significant, and it is worth exploring how decentralized science can be used to improve the way we conduct research and share knowledge.
 
-Intellectual property is a problem in TradSci. However, ownership of digital assets (such as scientific data or articles) is something Web3 does exceptionally using NFTs.
+**Participants in Web3**
 
-### Data storage
+The tools DeSci produces are easy to understand and can be explained without using technical jargon. For example, research objects are scientific papers that contain more information than a simple PDF. Imagine being able to interact with a research paper instead of just reading it. Imagine being able to see the code used to run simulations and running it yourself with a simple click of a button. Imagine being able to see every change, evolution, and contributor to the research over its lifetime, just like with Git. 
 
-Scientific data must be universally available and not hidden away behind paywalls. It must also be out of centralized control and able to survive cataclysmic events. Web3 file storage solutions such as IPFS, Arweave and Filecoin meet these requirements much better than traditional private servers. dClimate, for example, provides univbersal access to climate and weather data, including from weather stations and predictyive climate models.
+Publishing networks allow research to be published, reviewed, elevated, and replicated without input from a controlling authority or company. Imagine never having to pay to access a research object. Imagine researchers getting paid directly for their work instead of through an intermediary. Imagine a marketplace of people and entities that provide review, curation, and replication services, instead of an oligopoly of a few major conglomerates.
 
-### ...what else?
+**Contributors to Web3**
 
-Derp derp derp
+The people and institutions who have been brought on board through DeSci's tooling have a lot of experience and knowledge in different areas. They've been working on similar problems to those facing Web3 for a long time. DeSci is making tools that will make their lives easier and let them keep working on the hardest and most rewarding problems facing society.  Having economists and mathematicians, to social psychologists and historians, with centuries of knowledge from hundreds of thousands of academics can help build the foundation of the macro-systems we're trying to create. In doing this, we're inviting them into the trustless world to keep doing what they're good at: solving the most challenging and fulfilling problems society has to offer.
 
-## How does DeSci work?
 
-Derp derp derp
-How does it work? How is Ethereum/blockchain technology used?
+## Innovation
+
```

Comment:
Proposed change:

Decentralized science will require new ways of funding, creating, reviewing, crediting, storing, and disseminating knowledge. These novel developments will undoubtedly affect other realms as well. From rich on-chain research objects to IP-NFTs and everything in between, the development happening in DeSci will be pivotal to the digital world moving forward.